qml/plugins/RestartAudio.qml

```diff
@@ -0,0 +1,26 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+import Sailfish.Utilities 1.0
+import Nemo.DBus 2.0
+
+ActionItem {
+    //% "Audio"
+    title: qsTrId("sailfish-tools-he-audio")
+    //% "Restart"
+    actionName: qsTrId("sailfish-tools-bt-restart")
```

Comment:
bt?

Reply:
Ah, reuse...

Reply:
Yeap, reuse but to avoid misunderstanding.

-he- => header
-bt- => button

https://docs.sailfishos.org/Reference/I18n/I18n_Conventions/#platform-application-style-requirements